launch_ros/launch_ros/actions/node.py

```diff
@@ -235,7 +239,9 @@ def parse(cls, entity: Entity, parser: Parser):
         node_name = entity.get_attr('node-name', optional=True)
         if node_name is not None:
             kwargs['node_name'] = node_name
-        kwargs['package'] = parser.parse_substitution(entity.get_attr('pkg'))
+        package = parser.parse_substitution(entity.get_attr('pkg'), optional=True)
```

Comment:
This is the line where nightly/CI is failing.

Reply:
Yes, that's wrong. I'll fix it and check why it didn't fail before.

Reply:
Cool thanks, I've got it as part of this issue, so if you fix it could you comment there too? https://github.com/ros2/build_cop/issues/244